vaadin-testbench-core/src/main/java/com/vaadin/testbench/commands/ScreenshotComparator.java

```diff
@@ -26,7 +28,7 @@
 
 public class ScreenshotComparator {
 
-    private static Boolean supportsElementScreenshots = null;
+    private static Set<String> browsersWithoutElementScreenshot = new HashSet<>();
```

Comment:
Can this be accessed by multiple threads at the same time? Looks like there could be concurrency issues

Reply:
Done